template/.github/workflows/ci.yml

```diff
@@ -46,4 +46,24 @@ jobs:
 
     - name: Run type-checking
       run: poetry run invoke typing
-  
+
+  documentation-building:
+    name: Documentation Building
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
+        cache: 'poetry'
+
+    - name: Install dependencies
+      run: |
+        pip install invoke poetry
+        poetry install
+
+    - name: Build the documentation
+      run: TZ=UTC poetry run invoke build-docs
```

Comment:
Setting `TZ=UTC` is a workaround to avoid raising ValueError when running the CI using `act`. Please have a look at [this](https://github.com/nektos/act/issues/1853) similar issue.

Reply:
Interesting, that issue is about sphinx but I suspect that something we are using in MkDocs (Material) also relies on Babel, which appears to be the package causing drama.

If this were a test suite I think it might be more concerning but since it it just the documentation and is unlikely to have any significant impact on them I am happy with this work around.

However, I would like you to add a comment in this file to with your explanation for why this `TZ=UTC` environmental variable is being set for this job. Without understanding is could be targeted as an oddity for clean-up. Rather let them know why now, than have them lose time finding out later.

Reply:
Yes, Babel makes the same issue for MkDocs. Interesting point is that based on [this](https://github.com/ESKYoung/cookiecutter-machine-learning/commit/8cbfe602ccb6e561566b180bff380ab51e300e11) temporary fix on act, it's supposed to be fixed but it's not.

Reply:
I think the reason that above fix didn't work here is that we use poetry for managing the env variables.

Reply:
Added comments in [667b5db](https://github.com/ackama/django-template/pull/44/commits/667b5dbf20279fb165a8fbfc738c76db47ab5688)